sympy/sets/sets.py

```diff
@@ -1784,6 +1784,12 @@ def _complement(self, other):
             elif nums == []:
                 return None
 
+        elif isinstance(other, FiniteSet):
+            elms_unkown = FiniteSet(*[el for el in self if other.contains(el) not in (True, False)])
```

Comment:
Elements in `self` which return `Contains` object for `other.contains`. Does not the name sound right for that ?